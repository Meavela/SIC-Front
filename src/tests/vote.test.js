import React from 'react';
import renderer from 'react-test-renderer';
import Questions from '../components/list-question';
import Result from '../components/result';
import Navbar from '../components/navbar-component';
import { BrowserRouter } from 'react-router-dom';

it('test questions list', () => {
  const tree = renderer
    .create(<Questions />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('test results', () => {
  const tree = renderer
    .create(<Result />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('test navbar', () => {
  const tree = renderer
    .create(<BrowserRouter><Navbar /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});