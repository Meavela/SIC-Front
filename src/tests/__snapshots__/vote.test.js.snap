// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test navbar 1`] = `
<div>
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <span
      className="navbar-brand"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Votes
      </a>
    </span>
    <span
      className="navbar-brand"
    >
      <a
        href="/admin"
        onClick={[Function]}
      >
        Admin
      </a>
    </span>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        className="mr-auto navbar-nav"
        onKeyDown={[Function]}
      />
      <div
        className="navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rb-event-key="/login"
          href="/login"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Se connecter
        </a>
      </div>
    </div>
  </nav>
</div>
`;

exports[`test questions list 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-md-4 col-6"
    >
      <div
        className="list-group"
        onKeyDown={[Function]}
        style={
          Object {
            "marginTop": 200,
          }
        }
      />
    </div>
    <div
      className="col"
    >
      <div />
    </div>
  </div>
</div>
`;

exports[`test results 1`] = `<div />`;
